modules/osm/node.js

```diff
@@ -120,7 +120,13 @@ const prototype = {
 
             var nodeIds = {};
             resolver.parentWays(this)
-                .filter(way => osmShouldRenderDirection(this.tags, way.tags))
+                .filter(way => {
+                    // check for outbound one-way ways from stop signs
+                    const isOutbound = way.tags.oneway === 'yes' && way.nodes[0] === this.id;
```

Comment:
there are [other tags that imply `oneway`](https://github.com/openstreetmap/id/blob/develop/modules/osm/tags.js#L153), so it'd be better to use [some of the existing helper functions](https://github.com/openstreetmap/iD/blob/992c712c9f2b96227a38fadabed15d37fcbb0cf7/modules/osm/way.js#L145-L170). For example:
```suggestion
                    const isOutbound = way.isOneWayForwards() && way.nodes[0] === this.id;
```